src/components/FieldComment.tsx

```diff
@@ -2,21 +2,23 @@ import React from 'react'
 
 import Block from './Block'
 import InputString from './InputString'
+import { WithTranslation, withTranslation } from 'react-i18next';
 
 type FieldCommentProps = {
   value?: string
   onChange(value: string | undefined): unknown
   error: {message: string}
 };
 
-export default class FieldComment extends React.Component<FieldCommentProps> {
+class IFieldComment extends React.Component<FieldCommentProps & WithTranslation> {
   render() {
+    const t = this.props.t;
     const fieldSpec = {
-      doc: "Comments for the current layer. This is non-standard and not in the spec."
+      doc: t("Comments for the current layer. This is non-standard and not in the spec."),
```

Comment:
I think it would be better to use a shorter key for translation?

Reply:
For simplicity and maintainability, I'm using English as the default language. It's possible to use arbitrary keys, but the downsides outweigh the benefits, I think. The biggest benefits of using English as the translation key, I think, are: 
* familiarity for development (right now, I think most developers working on this project are mainly focused on English-based audiences)
* being able to pick up on changes - for example, if we use a different key structure (in this case, something like `fieldcomment.comment.docstring`), and a developer updates the meaning of this string, they'll only update the translation file of the language(s) that they know. It's easier to find a new string that hasn't been translated yet than it is to find a string that had its meaning changed. This can be prevented by making rules like "if you change the meaning of a string, it needs to be a new translation ID", but I think at this point, translation should be as unobtrusive to the people who are familiar with the codebase as possible.

Reply:
Fair enough. thanks.